### DISCUSSION TITLE
Filtering on column with a relation

### DISCUSSION THREAD

**Discussion by marcwitteveen** [Power-Components/livewire-powergrid#361] [Q&A]
Hello,

I have a datasource like below and would like to know how to add a column so I can search / filter on column bossid_to_crmid:id...I've experimented with various possibilities but without success. Please be so kind to push me into the correct direction.
```
public function datasource(): ?Builder
{
    return PA::with('bossid_to_crmid:id');
}
```

**Comment by dansysanalyst** [comment]
Hi @marcwitteveen,

You can see an example at [PowerGrid Demo](https://github.com/Power-Components/powergrid-demo) repository, more specifically at the  [DishesTable](https://github.com/Power-Components/powergrid-demo/blob/main/app/Http/Livewire/DishesTable.php) file. This repository has a PowerGrid fully configured.

Also, more information here:

https://livewire-powergrid.com/#/table/datasource?id=datasource
https://livewire-powergrid.com/#/table/column-filters?id=filter-by-relationship

PS: Slovenian translation is on the way! 😎

________
⭐ Enjoying PowerGrid? Please consider Starring the repository!
📢 Share PowerGrid with your dev-community, friends and colleagues: Tweet, write a blog, record review video. 
👥 PowerGrid is open-source, entirely developed and maintained by the author and the contributors.

**Reply by marcwitteveen** [reply]
I have looked at that solution, it works amazing when databases / tables are on the same server. Unfortunately I have a database on a different server and I am using Eloquent ORM to solve that problem, because on the Model I can set the database connection. The model logic works, however the filter and search does not and throws an exception in the generated SQL, saying it can not find the column I am filtering on.

**Comment by CrazyBoy49z** [comment]
#353 This problem is not with remote server
